tests/plugins/test_koji_promote.py

```diff
@@ -852,7 +852,6 @@ def validate_output(self, output, metadata_only, has_config,
             ])
             if base_from_scratch:
                 expected_keys_set.remove('parent_id')
-                'parent_id' not in docker
```

Comment:
Is there just a missing `assert` here?

Reply:
Yes, but it is unnecessary, the dict keys are tested below with
```python
assert set(docker.keys()) == expected_keys_set
```
which already makes sure `'parent_id'` is not in the `docker` dict.